parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetFileWriter.java

```diff
@@ -69,6 +72,20 @@
   public static final byte[] MAGIC = "PAR1".getBytes(Charset.forName("ASCII"));
   public static final int CURRENT_VERSION = 1;
 
+  // need to supply a buffer size when setting block size. this is the default
+  // for hadoop 1 to present. copying it avoids loading DFSConfigKeys.
+  private static final int DFS_BUFFER_SIZE_DEFAULT = 4096;
```

Comment:
We weren't exposing a way to set this when we create the file anyway, so this isn't removing an option that users had. It is setting an option that the library had and wasn't using. Hard-coding it to the value we would always use anyway. I need to have a value for this because there isn't a method call that allows us to set the block size and default the buffer size. This default is very stable so I think this is perfectly safe.